libsolidity/interface/CompilerStack.cpp

```diff
@@ -416,6 +417,29 @@ void CompilerStack::importASTs(map<string, Json::Value> const& _sources)
 	storeContractDefinitions();
 }
 
+void CompilerStack::importEvmAssemblyJson(map<string, Json::Value> const& _sources)
+{
+	solAssert(_sources.size() == 1, "");
```

Comment:
How come our indents are inconsistent? They seem to vary between 4 and 8 spaces.

Reply:
I think that is just a visualisation problem on GitHub. The real code contains just one tab.